include/grpc/grpc_security.h

```diff
@@ -789,11 +789,13 @@ GRPCAPI grpc_tls_key_materials_config* grpc_tls_key_materials_config_create(
     void);
 
 /** Set grpc_tls_key_materials_config instance with provided a TLS certificate.
-    config will take the ownership of pem_root_certs and pem_key_cert_pairs.
     It's valid for the caller to provide nullptr pem_root_certs, in which case
     the gRPC-provided root cert will be used. pem_key_cert_pairs should not be
-    NULL. It returns 1 on success and 0 on failure. It is used for
-    experimental purpose for now and subject to change.
+    NULL.
+    The ownerships of |pem_root_certs| and |pem_key_cert_pairs| remain with the
+    caller and the caller is free to deallocate it immediately after the call.
```

Comment:
nit. The sentence - "the caller is free..."  seems redundant especially with "The ownerships of |pem_root_certs| and |pem_key_cert_pairs| remain with the caller".